src/Microsoft.Extensions.Testing.Abstractions/DIA/DiaDataSource.cs

```diff
@@ -0,0 +1,13 @@
+// Copyright (c) .NET Foundation and contributors. All rights reserved.
```

Comment:
Yeah, these are also pretty minimal. It's a lot of files but most are just COM Interop interfaces